backend/src/database/entities/emcr/emcr-personnel.entity.ts

```diff
@@ -126,6 +147,9 @@ export class EmcrPersonnelEntity {
       ...personnelData,
       dateApplied: this.dateApplied ?? '',
       dateApproved: this.dateApproved ?? '',
+      firstChoiceFunction: this.firstChoiceSection,
```

Comment:
Sorry I missed this earlier - do we need to set this to an empty string in the RO  since in most cases this data will not exist in the beggining?

Reply:
It wasn't needed in the bcws entity, so maybe not

Reply:
Because it can be undefined